.travis.yml

```diff
@@ -28,9 +28,7 @@ matrix:
     - name: "3.8"
       python: 3.8
     - name: "3.9"
-      python: 3.9-dev
-  allow_failures:
-    - python: 3.9-dev
+      python: 3.9
```

Comment:
Travis CI doesn't yet have `3.9` so we need to stick with `3.9-dev` for now.

Here's the tracking issue to +1:

* https://travis-ci.community/t/python-3-9-0-build/10091?u=hugovk